src/sm.c

```diff
@@ -154,7 +155,7 @@ void sm_open_end(SM_HANDLE sm, bool success)
     else
     {
         /*Codes_SRS_SM_02_041: [ If state is not SM_OPENING then sm_open_end shall return. ]*/
-        LONG state = InterlockedAdd(&sm->state, 0);
+        LONG state = interlocked_add(&sm->state, 0);
```

Comment:
>LONG [](start = 8, length = 4)

int32_t

Reply:
here and everywhere probably

---
In reply to: [501161065](https://github.com/Azure/azure-c-util/pull/66#discussion_r501161065) [](ancestors = 501161065)

Reply:
Fixed.

---
In reply to: [501161280](https://github.com/Azure/azure-c-util/pull/66#discussion_r501161280) [](ancestors = 501161280,501161065)